### DISCUSSION TITLE
How does bambu handle secondary/supplementary alignmets, low MAPQ, and MAPQ=0 (reads that align equally well to 2+ loci)?

### DISCUSSION THREAD

**Discussion by bernardo-heberle** [GoekeLab/bambu#343] [General]
Hello,

I was wondering how Bambu handles these three alignment cases:

1. Secondary and Supplementary alignments in a bam file? Does Bambu do anything with those or just ignores them?

2. Low MAPQ score? Is there a threshold where Bambu stops taking a read alignment into account if the MAPQ is too low?

3. MAPQ=0? From my understanding when a read aligns equally well to two or more different loci in the reference genome Minimap2 gives it a MAPQ=0 and randomly assigns the read to one of the loci. Does Bambu take those alignments into account or filters them out?

Thank you for your time.

**Comment by andredsim** [comment]
Hi,

Thanks for your questions, see the answer to each below:

1.
Secondary alignments - Ignored by Bambu
Supplementary alignments - Included by Bambu and treated as normal reads.

2. Currently Bambu does not filter by MAPQ score. If you have particular requirements based on your aligner, we would recommend filtering the bam file by MAPQ before providing it to Bambu.

3. If this case occurs, Bambu would only use the arbitrarily assigned primary alignment. This is probably something we should keep in mind for future releases to see if we can improve on this.

I hope this answers your questions about the current version of Bambu, let me know if you have anymore.

-Andre
